init-tools.sh

```diff
@@ -42,7 +43,7 @@ esac
 
 if [ ! -e $__PROJECT_JSON_FILE ]; then
     if [ -e $__TOOLRUNTIME_DIR ]; then rm -rf -- $__TOOLRUNTIME_DIR; fi
-
+    echo "Running: $__scriptpath/init-tools.sh" > $__init_tools_log
     if [ ! -e $__DOTNET_PATH ]; then
         echo "Installing dotnet cli..."
```

Comment:
Consider also logging the remote path from which it gets the dotnet CLI and the local path where it installs it.  In init-tools.cmd, this is done as:
`echo Installing '%DOTNET_REMOTE_PATH%' to '%DOTNET_LOCAL_PATH%' >> %INIT_TOOLS_LOG%`